### DISCUSSION TITLE
Updating cart item quantity when stock is too low

### DISCUSSION THREAD

**Discussion by Web10-Joris** [duncanmcclean/simple-commerce#944] [Feature Requests]
I'm not sure if this functionality is already built-in, but in my current setup, when a user tries to update the quantity of a cart item using the `{{ sc:cart:updateItem }}` form to a value higher than the available stock, the page simply refreshes without making any changes to the cart. Is it possible to enhance this behavior by adding the maximum available amount of that product to the cart and sending a notification to the user, such as: "We only have X items left of Product"?

**Comment by duncanmcclean** [comment]
Somehow I missed the notification for this, sorry! 

>  Is it possible to enhance this behavior by adding the maximum available amount of that product to the cart and sending a notification to the user, such as: "We only have X items left of Product"?

Unfortunately, this isn't possible at the moment. I'll move this discussion into the "Feature Request" category so it's considered in the future. I'd welcome a PR to change this behaviour for v6.

**Comment by Web10-Joris** [comment]
No problem! Thanks for the response. When I have time and figured out how to add this I will create a PR.
